web/src/components/core/AlertOutOfSync.tsx

```diff
@@ -72,7 +72,7 @@ export default function AlertOutOfSync({ scope, ...alertProps }: AlertOutOfSyncP
     if (missingScope) return;
 
     return client.onEvent((event) => {
-      event.type === `${scope}Changed` && setActive(event.clientId !== client.id);
+      event.type === `${scope}Changed` && setActive(event.client_id !== client.id);
```

Comment:
I actually expect `clientId` here. I though they were using something like `#[serde(rename_all = "camelCase")]` in Rust part, but most probably I'm missing something.

I even would say that this worked with `clientId` form when I tested it in previous PRs.

Reply:
Actually, the client id was wrongly reported in the `StorageChanged` event.